pkg/kv/kvserver/client_raft_test.go

```diff
@@ -6183,6 +6183,12 @@ func TestRaftForceCampaignPreVoteCheckQuorum(t *testing.T) {
 	logStatus(initialStatus)
 	t.Logf("n1 is leader in term %d", initialStatus.Term)
 
+	// Wait for any pending config changes to be finalized. Otherwise, the forced
+	// campaign will fail and not retry.
```

Comment:
Do we need it to win? The comment below says it's enough that it bumps the term.